test/e2e/features/step_definitions/steps/approveOrganisation.steps.js

```diff
@@ -20,10 +20,15 @@ defineSupportCode(function ({ Given, When, Then }) {
 
     When(/^I navigate to EUI Approve Organisation Url$/, { timeout: 600 * 1000 }, async function () {
         await browser.get(config.config.baseUrl) ;
-        await browser.driver.manage()
+        if (!process.env.TEST_URL.includes('demo')){ //Do not delete cookies for demo env.
```

Comment:
If cookies are not cleared before starting a scenario, in current scenario we need to check if there is a login step coming up next does not fail scenario. Are the affects of this change addressed?

Reply:
@sreekanthpuligadda This was verified prior to making the change. For approve-org, I commented out logout for one of  the scenario and tested. The next scenario it executed was successful. The cookie persistence is only for demo env (inside if condition). I believe this is a known issue with demo env.  Is there any other test i would need to run and check?

Reply:
this is just workaround/additional code anyways it wont break any FT's on master/nightly/PR pipelines and demo doesn't have any pipeline itself, it helps when we run tests local pointing to demo and Rohith ran all test suite and all passed, there is no scenario without a login page if cookie is cleared,